src/views/partials/content.ejs

```diff
@@ -1,13 +1,15 @@
 <main class="products-wrapper">
   <%products.forEach( function( prd ){ %>
-  <article class="product">
-    <figure class="figure">
-      <img src=<%= prd.product_image %> alt=<%= prd.product_name %> >
-      <figcaption class="product-content">
-        <p class="product-price">From $<%= prd.product_price%></p>
-        <h4 class="product-title"><%= prd.product_name %></h4>
-      </figcaption>
-    </figure>
-  </article>
+    <article class="product">
+      <figure class="figure">
+        <a href="/product/<%= prd.product_name %>" title = <%= prd.product_name %> target="_self">

```

Comment:
```suggestion
        <a href="/product/<%= product.product_name %>" title = <%= product.product_name %> target="_self">
```
I think it's a good opportunity to rename from `prd` to `product`. Please update.

Reply:
Done